test/check-application

```diff
@@ -132,8 +130,10 @@ class TestApplication(testlib.MachineCase):
         b.wait_present('.modal-dialog div:contains("io.podman.ErrorOccurred")')
         b.click(".modal-dialog div .alert .close")
         b.wait_not_present(".modal-dialog div .alert")
+        self.confirm_modal("btn-ctr-cancel-commit")
 
         # delete image busybox that hasn't been used
+        b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
```

Comment:
Is that an accidental copy&paste, or deliberate? There's now a wait_present *after* this click.

Reply:
Without it it fails, because the image row will not be expanded for the delete button to be visible (line 139), but I noticed now that there is a second toggle before the button click (line 138), so I will remove this and the second and the state of the row should be expanded